modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/utils/ConfigUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *   * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.synapse.config.xml.endpoints.utils;
+
+import org.apache.synapse.SynapseConstants;
+
+public class ConfigUtils {
+
+    public static String fetchEnvironmentVariables(String parameter) {
+        String injectedValue = parameter;
+        if (parameter.contains(SynapseConstants.SYSTEM_VARIABLE_PREFIX)) {
```

Comment:
shall we use startsWith() instead of contains?

Reply:
Resolved https://github.com/apache/synapse/pull/55/commits/60c9ddc3be1a79d06a8e2fa597ba340d2abbf209